app/actions/index.js

```diff
@@ -125,7 +168,7 @@ export const clearRemoteSurveys = () => (dispatch, getState) =>
     type: types.CLEAR_REMOTE_SURVEYS
   });
 
-export const fetchRemoteSurvey = (id, url) => (dispatch, getState) => {
+export const fetchRemoteSurvey = (id, url, target) => (dispatch, getState) => {
```

Comment:
How do we know if the OSM data that's previously been synced is still correct for the surveys being added?

With the changes coming with #142, all OSM data is removed each time before it is synced. It seems like we need a way to verify with the mobile app whether its current OSM data is valid for the active surveys.

Before #142 syncing again wasn't too big a deal because if the OSM data didn't change subsequent syncs take very little time, as there's nothing to sync.